front/webpack.config.ts

```diff
@@ -101,6 +101,18 @@ module.exports = {
                     fullySpecified: false
                 }
             },
+            {
+                test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+                use: [
+                    {
+                        loader: 'file-loader',
+                        options: {
+                            name: '[name].[ext]',
+                            outputPath: 'fonts/'
+                        }
+                    }
+                ]
+            },
```

Comment:
You should remove the file-loader. It is deprecated in Webpack 5 and replaced by the notion of assets.